src/components/footer/index.jsx

```diff
@@ -18,7 +18,6 @@ class Footer extends React.Component {
       <footer className="footer-container">
         <div className="footer-body">
           <img src={getLink(logo)} />
-          <p className="docsite-power">website powered by docsite</p>
```

Comment:
Why delete this?

Reply:
I see similar websites do not have this sentence .
I can roll back this submission if necessary.

Reply:
I think we need to keep it

Reply:
received